conans/test/command/json_output_test.py

```diff
@@ -22,7 +22,8 @@ def test_simple_fields(self):
         self.assertFalse(my_json["error"])
         self.assertEquals(my_json["installed"][0]["recipe"]["id"], "CC/1.0@private_user/channel")
         self.assertFalse(my_json["installed"][0]["recipe"]["dependency"])
-        self.assertTrue(my_json["installed"][0]["recipe"]["cache"])
+        self.assertTrue(my_json["installed"][0]["recipe"]["exported"])
```

Comment:
@lasote Unfortunately adding recipe as exported to recorder in `conan create` makes it to be marked as exported but not as cache. I think this is OK, but changes behavior

Reply:
The `ActionRecorder` retrieves only the last `action` ([see here](https://github.com/conan-io/conan/pull/3809/files#diff-cee7c925d6aa4550a1c0b57434aaa2c4R159)). It's something I came accross when working in the `cpp_info` issue and I wonder if it makes sense to show all the actions related to a recipe in the `json` output (not to be done in this PR, but maybe it is worth considering it).

Reply:
It was getting the latest action because until now it was the only needed, but it would be very very easy to pass all the actions to the `get_doc_for_ref` and use all to calculate the "export" field and maybe `cache == not download` or something similar, right?

Reply:
By the way, maybe we have to remove the "cache" attribute, because it only means `download=False`